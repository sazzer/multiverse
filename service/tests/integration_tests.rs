mod data;
mod service;
mod tests;
