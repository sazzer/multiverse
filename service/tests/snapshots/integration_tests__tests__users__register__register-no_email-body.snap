---
source: tests/integration_tests.rs
expression: response.to_json().unwrap()
---
{
  "fields": {
    "email_address": {
      "title": "The required field was missing",
      "type": "tag:multiverse,2020:problems/validation_error/missing"
    }
  },
  "status": 422,
  "title": "A validation error occurred",
  "type": "tag:multiverse,2020:problems/validation_error"
}
